extensions/stackdriver/common/utils.cc

```diff
@@ -35,6 +36,23 @@ const std::string getContainerName(
   return kIstioProxyContainerName;
 }
 
+const std::string getNodeID(
```

Comment:
const std::string is a tautology. just a string is fine.

Reply:
done.